source/dub/generators/generator.d

```diff
@@ -121,11 +121,13 @@ class ProjectGenerator
 	protected {
 		Project m_project;
 		NativePath m_tempTargetExecutablePath;
+		PackageManager m_packageMan;
```

Comment:
I think that's the reason I didn't do it in the first place - Having the `PackageManager` in the generator is a bit icky.